include/openenclave/internal/sgxtypes.h

```diff
@@ -25,10 +25,16 @@ OE_EXTERNC_BEGIN
 #define SGX_FLAGS_PROVISION_KEY 0x0000000000000010ULL
 #define SGX_FLAGS_EINITTOKEN_KEY 0x0000000000000020ULL
 
-#define SGX_XFRM_LEGACY 0x0000000000000003ULL
-#define SGX_XFRM_AVX 0x0000000000000006ULL
+#define SGX_XFRM_LEGACY                                                     \
+    0x0000000000000003ULL /* Legacy XFRM which includes basic feature bits  \
+                             required by SGX i.e. X87 (bit 0) and SSE state \
+                             (bit 1) */
+#define SGX_XFRM_AVX                                                           \
+    0x0000000000000006ULL /* AVX XFRM which includes AVX (bit 2) and SSE State \
+                             (Bit 1) required by AVX */
 #define SGX_XFRM_AVX512 0x00000000000000E6ULL
```

Comment:
FYI, Intel also labels this as unsupported in their SDK.

Reply:
Yes, noticed that and adding comments in our code as well

Reply:
Done